app/utils/commands_map.py

```diff
@@ -1,9 +1,19 @@
+<<<<<<< HEAD
+from .commands import (
+    input_prop_equals,
+    input_prop_in,
+    input_prop_in_as,
+    allow_full_access,
+)
+=======
 from .commands import input_prop_equals
 from .commands import input_prop_in
 from .commands import input_prop_in_as
+>>>>>>> main
```

Comment:
Is this a merge conflict? Can you take a look at it?

Reply:
> 

Fixed.

Reply:
Great, 👍🏾 .